packages/callbacks/example/preview-resolutions/lib/main.dart

```diff
@@ -0,0 +1,172 @@
+import 'package:flutter/material.dart';
+import 'dart:ffi';
+import 'dart:isolate';
+
+import 'package:ffi/ffi.dart';
+import 'package:tizen_interop/6.5/tizen.dart';
+import 'package:tizen_interop_callbacks/tizen_interop_callbacks.dart';
+import 'package:tizen_log/tizen_log.dart';
+
+MyAppState? myAppState;
+
+void main() {
+  runApp(const MyApp());
```

Comment:
Why don't you make the examples into a single two-paged app?

The `api-version` value in `tizen-manifest.xml` doesn't affect the plugin compilation unless the plugin depends on Tizen native headers so I think it can be set to either 4.0 or 6.5. What happens if you call a Tizen 6.5 API in Dart code on a Tizen 6.0 (or older) device?

Reply:
The idea was to make the examples as clean and simple as possible. I can combine them into one if you prefer to follow typical schema of a single app in `example` directory.

Calling a tizen_interop_callbacks code for a callback that does not exists on current platform is safe - you can register it, as all necessary data is copied (with substitutions) into generated code. But would even work if use was able to execute the wrapper/proxy callback - but that would be hard because the related Native API that sets the callback to be called is not available.

Reply:
Thanks for the explanation. My preference is to make them into a single app using, for example, either (list/buttons & navigator) or tabs to separate the examples within the app. It's up to your choice though.

> But would even work if use was able to execute the wrapper/proxy callback - but that would be hard because the related Native API that sets the callback to be called is not available

I don't quite understand this part. The user can register a callback and can call `tizen.device_add_callback`, but the callback just never gets called?